### PR TITLE
Add AWS Extension Node Documentation

### PR DESCRIPTION
**Pull request by zkayyali812**
PR to add documentation about how to add extension nodes to AAP on AWS

### CODE REVIEW

File: stories/topics/ref-aap-aws-add-extension-nodes.adoc [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+By default, only the foundation stack is deployed with 2 {ControllerName} nodes and 1 {HubName} node. Execution nodes can be added to scale out {PlatformNameShort} from {Market}.
+Follow these steps to add execution nodes into your {PlatformNameShort} from {Market} environment:
+
+. Open the {ControllerName} Launch Configuration
+.. Navigate to the AWS CloudFormation UI and click the *Resources* tab. Ensure *Tree View* is selected in the top right corner.
+.. Expand *controller*, then *autoscale-group*, then *Launch Config*.
+.. Click the Launch Config Physical ID to open the launch configuration in a new tab.
+. Create the extension node launch configuration
+.. In the Launch configuration tab, ensure the {ControllerName} launch configuration is selected, then click *Copy to launch template*, then *Copy selected*.
+.. Replace the *New launch template name* with the name 'extension-group-template' and ensure the checkbox for *Create an Auto Scaling group using the new template* is checked. Click Copy.
```
Comment (prat98) [inline stories/topics/ref-aap-aws-add-extension-nodes.adoc:14]: Making this bold would make it easier to find
```suggestion
.. Replace the *New launch template name* with the name *extension-group-template* and ensure the checkbox for *Create an Auto Scaling group using the new template* is checked. Click Copy.
```

**Comment by ariordan-redhat** [comment]
@zkayyali812 
I created a PR to your branch for changes to the procedure module.

**Review by ariordan-redhat — CHANGES_REQUESTED** [review]
Need to include the assembly in stories.adoc
Suggested changes to the topic file in a PR to your branch.

File: stories/assembly-aap-aws-extension-nodes.adoc [hunk 2]
```diff
@@ -0,0 +1,8 @@
+ifdef::context[:parent-context: {context}]
```
[the remark this answers is not in the document]
Comment (zkayyali812) [reply stories/assembly-aap-aws-extension-nodes.adoc:1]: Great catch! I just added this to the stories.adoc in the most recent commit

File: stories/topics/ref-aap-aws-add-extension-nodes.adoc [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+.Create the extension node launch configuration
+
+. In the *Launch configuration* tab, ensure the {ControllerName} launch configuration is selected.
+Click *Copy to launch template*, then *Copy selected*.
+. Replace the *New launch template name* with *extension-group-template*.
+Ensure the checkbox for *Create an Auto Scaling group using the new template* is checked.
+Click btn:[Copy].
+. For the Auto Scaling group name, enter `extension-group`.
+. Under *Launch Template*, click btn[Create a launch template version] to create a new version of the launch template.
```
Comment (ariordan-redhat) [inline stories/topics/ref-aap-aws-add-extension-nodes.adoc:23]:
```suggestion
. Under *Launch Template*, click btn:[Create a launch template version] to create a new version of the launch template.
```

**Review by ariordan-redhat — CHANGES_REQUESTED** [review]
Fix asciidoc formatting for UI elements

**Review by ariordan-redhat — APPROVED** [review]
LGTM

**Pull request merged**